src/bci_build/package/base/README.md.j2

```diff
@@ -9,27 +9,27 @@
 
 {{ pretty_bci }} provide open, flexible,
 and secure container images. The images include container environments based on
-{{ image.os_version.pretty_distribution_base_name }} and are available at no cost, they can be freely
-re-distributed, and they are supported across different environments.
+{{ image.os_version.pretty_distribution_base_name }} and are available at no cost, may be freely
+redistributed, and are supported across a large variety of environments.
 
-The Base Container Image is an image used as a foundation for many {{ short_bci }}s. The
-image is intended to be extended for further use, such as a development or a
-testing environment.
+The Base Container Image serves as a foundation for many {{ short_bci }}s. It is designed to
```

Comment:
This would read as:

The Base Container Image serves as a foundation for many SUSE Linux BCIs.

or if you expand the name:

The Base Container Image serves as a foundation for many SUSE Linux Base Container Images.

It is rather confusing to call the base image, base container image because it is in theory:

SUSE Linux BCI (Base Container Image) Base Container Image.

Reply:
works for me. updated.